tools/mkmapdata/mkmapdata.sh

```diff
@@ -356,13 +349,18 @@ case $hostname in
      mpi=uni
      mpitype="mpiuni"
   fi
-  module load esmf-${esmfvers}-ncdfio-${mpi}-O
-  if [ -z "$ESMFBIN_PATH" ]; then
-     ESMFBIN_PATH=`grep ESMF_APPSDIR $ESMFMKFILE | awk -F= '{print $2}'`
-  fi
   if [ -z "$MPIEXEC" ]; then
      MPIEXEC="mpiexec_mpt -np $REGRID_PROC"
   fi
+
+  module load ncarenv/1.3
```

Comment:
Is it critical to use specific versions of software packages? This can be something difficult to stay up to date with.

Reply:
This is the combination of module loads that @bekozi determined we needed for ocgis to run. I can test by trial-and-error whether removing the version numbers will continue to work, unless @bekozi advises against it.

Reply:
@bekozi, before attempting to remove specific version numbers by trial-and-error, I returned to my previously working script on cheyenne:
`/glade/work/slevis/ocgis_work/subset_1x1_20191213.sh`
and found that it now fails with this error in the PET0.ESMF_LogFile:
`20191229 114147.072 ERROR  PET0 ESMF_Array.F90:3962 ESMF_ArrayWrite Unable to write to file  - Internal subroutine call returned Error`
`20191229 114147.073 ERROR  PET0 ESMF_IOScrip.F90:367 ESMF_OutputWeight File Unable to write to file  - Internal subroutine call returned Error`
`20191229 114147.073 ERROR  PET0 ESMF_IOScrip.F90:136 ESMF_SparseMatrix Write Unable to write to file  - Internal subroutine call returned Error`
`20191229 114147.073 ERROR  PET0 ESMF_Field_C.F90:1269 f_esmf_regridstorefile Unable to write to file  - Internal subroutine call returned Error`
It still writes out the same subset file as before:
`/glade/work/slevis/ocgis_work/subsets/spatial_subset_191209.nc`
but fails to generate the weight file that it had previously:
`/glade/work/slevis/ocgis_work/weights/weights_191209_after_subsetg_1x1.nc`

I tried two things that didn't help. 
In `/glade/work/slevis/git_ocgis/ocgis` and 
in `/glade/work/slevis/git_esmf/esmf` I typed
`git fetch`
`git pull`
to get caught up with the latest codes.

What else am I missing?

Reply:
Turns out the job fails when the weight file already exists. I'm surprised that I had not run into this before :-)  Thanks for troubleshooting with me, @bekozi. Reminder for you to put a more explanatory error message in the code about this.

Reply:
Replaced specific module loads with generic module loads now.